src/watch.ui/Config.Designer.cs

```diff
@@ -356,6 +370,7 @@ private void InitializeComponent()
         private MetroFramework.Controls.MetroLabel metroLabel6;
         private System.Windows.Forms.ListBox LogList;
         private MetroFramework.Controls.MetroToggle metroToggle1;
+        private MetroFramework.Controls.MetroCheckBox chkWatchSubdirectories;

```

Comment:
Looks awesome! My only nitpick is the naming :) I suggest we don't prefix with chk/btn/cmb/etc as we should try to avoid hungarian notation as per guidelines in https://msdn.microsoft.com/en-us/library/ms229045.aspx

What do you think?